### ISSUE TITLE
Chose and implement other footprint methods

### ISSUE THREAD

**Issue by AlexDo1** [KIT-HYD/bridget#16]
Here is a table of important footprint models.

Choose an interesting model for implementation (analytical?, data availability) and then implement the method.

![image](https://user-images.githubusercontent.com/23619690/101639784-01ebbd80-3a30-11eb-85f6-2c40bd61a7b5.png)

_Originally posted by @AlexDo1 in https://github.com/KIT-HYD/bridget/pull/15#discussion_r539341083_

**Comment by mmaelicke** [comment]
@sihassl anything familiar to you here?
Is there something that definitely has to be implemented in the eddy package?

**Comment by sihassl** [comment]
Kormann and Meixner (2001) and Kljun (2002) for now. (and of course at some point all of them! :))

**Comment by AlexDo1** [comment]
Do you know if this is an updated/preferred model by Kljun or is it just an entirely different thing?

https://gmd.copernicus.org/articles/8/3695/2015/

**Comment by mmaelicke** [comment]
the Kljun 2015 is a 2D model, 2002 is a 3D model and Kljun 2004b is a 1D model. I guess that already makes them quite different.

**Comment by AlexDo1** [comment]
This is true, I was just wondering if the oldest model from Kljun is still the "best" or if they maybe released an improved version.
But these really seem to be completely different models.

**Comment by sihassl** [comment]
I just learned recently that Kljun is well-used, at least at UFZ. But I can ask which version.
